rolling/distribution.yaml

```
@@ -4907,6 +4907,11 @@ repositories:
       url: https://github.com/ros/robot_state_publisher.git
       version: rolling
     status: maintained
+  robotont_msgs:
+    doc:
+      type: git
+      url: https://github.com/robotont/robotont_msgs
```

```suggestion
      url: https://github.com/robotont/robotont_msgs.git
```

rosdistro repository urls use explicit suffixes to denote the vcs used.